imgui.go

```diff
@@ -686,3 +686,62 @@ func IsItemHovered() bool {
 func IsKeyPressed(key int) bool {
 	return C.iggIsKeyPressed(C.int(key)) != 0
 }
+
+// Columns calls ColumnsV(count, label, ColumnsFlagsNone).
+func Columns(count int, label string) {
+	ColumnsV(count, label, ColumnsFlagsNone)
+}
+
+// ColumnsV creates a column layout of the specified number of columns.
+func ColumnsV(count int, label string, flags int32) {
+	labelArg, labelFin := wrapString(label)
+	defer labelFin()
+	C.iggBeginColumns(C.int(count), labelArg, C.int(flags))
+}
+
+// NextColumn next column, defaults to current row or next row if the current row is finished.
+func NextColumn() {
+	C.iggNextColumn()
+}
+
+// ColumnIndex get current column index.
+func ColumnIndex() int {
+	return int(C.iggGetColumnIndex())
+}
+
+// ColumnWidth calls ColumnWidthV(-1).
+func ColumnWidth() int {
+	return ColumnWidthV(-1)
+}
+
+// ColumnWidthV get column width (in pixels). pass -1 to use current column.
+func ColumnWidthV(index int) int {
+	return int(C.iggGetColumnWidth(C.int(index)))
+}
+
+// SetColumnWidth sets column width (in pixels). pass -1 to use current column.
+func SetColumnWidth(index int, width float32) {
+	C.iggSetColumnWidth(C.int(index), C.float(width))
+}
+
+// ColumnOffset calls ColumnOffsetV(0)
+func ColumnOffset() float32 {
+	return ColumnOffsetV(0)
```

Comment:
The default argument is `-1` (also update comment)

Reply:
Ohh man sorry about that @dertseha, i feel dumb.

Amended the previous commit with the corrected default + comment

Reply:
heh, all is good, things happen - that's the beauty of code reviews - finding stuff early, rather than later :)